nodejs/10/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM        node:dubnium
+FROM        node:dubnium-jessie
```

Comment:
please use `node:10.15.3-jessie` as 10.16 brings in different base version of `npm`